lib/matplotlib/backends/backend_ps.py

```diff
@@ -1452,6 +1452,10 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     # multiple
     if sys.platform == 'win32': precmd = '%s &&'% os.path.splitdrive(tmpdir)[0]
     else: precmd = ''
+    #Replace \\ for / so latex does not think there is a function call
+    latexfile = latexfile.replace("\\", "/")
```

Comment:
No: you can also specify all slashes as backlashes: the name is passed to a program and either used as filename with backlashes but no ~ allowed or as a latex input with `/` as path separator which can contain `\string~` but no backlashes.